src/public/modules/forms/admin/directives/settings-form.client.directive.js

```diff
@@ -64,11 +66,20 @@ function settingsFormDirective(
         $scope.submissionLimitUnlimited = null
 
         $scope.currentResponsesCount = 0
-        Submissions.count({ formId: $scope.myform._id }).then(
-          (responsesCount) => {
-            $scope.currentResponsesCount = responsesCount
-          },
+        $q.when(
+          AdminSubmissionsService.countFormSubmissions({
+            formId: $scope.myform._id,
+          }),
         )
+          .then((responsesCount) => {
+            $scope.currentResponsesCount = responsesCount
+          })
+          .catch((error) => {
+            Toastr.error(
+              `Unfortunately, there was an error in displaying the number of responses. Please refresh and try again!`,
+            )
+            console.log(error)
```

Comment:
remove console log?

Reply:
i think the console log here has value - in angular, we log errors because we want to be able to see the stack trace. if we remove the console log here, we see generic error messages without context, which isn't helpful for troubleshooting.